<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{     
     background: qlineargradient(
   x1:0, y1:0, x2:0, y2:1,
   stop: 0.1 #999999,
	stop:0.5 #505050,
   stop:1 black
 );
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="settingsGroupsTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>42</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>42</number>
       </attribute>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Neue Zeile</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>Fader</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>Collection</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>Audio devices</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>AutoDJ</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>Playlists</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>File browser</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="pages">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Fader settings</string>
         </property>
        </widget>
        <widget class="QSlider" name="faderEndSlider">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>160</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>12</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Start fading before song end</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>371</width>
           <height>16</height>
          </rect>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="faderEndLabel">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>80</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>12s</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Fader Time</string>
         </property>
        </widget>
        <widget class="QLabel" name="faderTimeLabel">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>130</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>12s</string>
         </property>
        </widget>
        <widget class="QSlider" name="faderTimeSlider">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>160</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>12</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkAutoCue">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>321</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Skip silent at start (AutoCue)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkSkipSilentEnd">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>321</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Skip silent at the end</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="toolTip">
            <string/>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>-1</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTreeView" name="collectionsTreeView"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QCheckBox" name="chkMonitor">
                <property name="toolTip">
                 <string>Scan for new files and update the music collection</string>
                </property>
                <property name="text">
                 <string>Watch folders for changes</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushScanNow">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Scan Now</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Monitor Player</string>
         </property>
        </widget>
        <widget class="QComboBox" name="monitorOutputDevice">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>201</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>AutoDJ settings</string>
         </property>
        </widget>
        <widget class="Line" name="line_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>371</width>
           <height>16</height>
          </rect>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSpinBox" name="minTracks">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Minimum Tracks</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="countDJ">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>DJ Count</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_5">
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>371</width>
           <height>16</height>
          </rect>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkAutoRemove">
         <property name="geometry">
          <rect>
           <x>7</x>
           <y>74</y>
           <width>211</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Auto Remove Played Tracks</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Playlist Settings</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_6">
        <widget class="QLineEdit" name="txtBrowserRoot">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>211</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Browser Root</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>242</x>
           <y>47</y>
           <width>50</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
